CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## 5.1.1 / 2023-06-22
+
+## What's Changed
+* add default 1024 max stake limit for querying UIDs with vpermit. by @ifrit98 in https://github.com/opentensor/bittensor/pull/1379
```

Comment:
Are these mentions here useful, given that the validator no longer live in this repository ?